definitions/schemaDefinition.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
could add something to the `README` explaining what this file is and how to use it

Reply:
Can I add this? 



### Schema Definition

The repo contains 'definitions' folder with a schema definition json which can be imported into ts files like so:

```
import _schema from '<path>/definitions/schemaDefinition.json'. 
const schema = _schema as AnonCredsSchema
```

The schema definition can be posted to `/schemas` to register it. Upon a successful call a response is returned with an 'id' property which refers to this schema and can be used to refer to the schema when creating a new credential definition.
(Note: when registering a schema one needs to have an instance of ipfs running or be connected to global ipfs.)

Reply:
Another attempt: 


### Schema Definition

The repo contains 'definitions' folder with a schema definition json which can be imported into ts files like so:

```
import _schema from './definitions/schemaDefinition.json'
const schema = _schema as AnonCredsSchema
```

The schema gives us a blueprint for providing data for a particular check. It requires these fields:

- checkName => what sort of check this is (e.g. NASDAQ, Rev >500, Production Capacity etc)
- companyName => name of the company that is being checked (max 200 characters)
- companiesHouseNumber => unique identifier for companies provided by Companiess HOuse (max 8 characters - alphanumeric)
- issueDate => date when this check was issued (timestamp)
- expiryDate => date when this check expires (timestamp)

The schema definition can be posted to `/schemas` to register it. Upon a successful call a response is returned with an 'id' property which refers to this schema and can be used to refer to the schema when creating a new credential definition like so:

```
{
  "tag": "myTestDefinition",
  "schemaId": "ipfs://example",
  "issuerId": "did:key:exampleDidKey"
}
```

A credential definition can then be used to issue a credential which contains both information about an issuer of a credential and the check itself.
(Note: when registering a schema one needs to have an instance of ipfs running or be connected to global ipfs.)

Reply:
yeah that's great 👍